Packs/Reco/Integrations/Reco/Reco_test.py

```diff
@@ -785,3 +858,13 @@ def test_change_alert_status(requests_mock, reco_client: RecoClient) -> None:
     res = reco_client.change_alert_status(alert_id=str(alert_id),
                                           status=status)
     assert res == {}
+
+
+def test_get_user_context_by_email(requests_mock, reco_client: RecoClient) -> None:
+    raw_result = get_random_user_context_response()
+    requests_mock.post(
+        f"{DUMMY_RECO_API_DNS_NAME}/asset-management", json=raw_result, status_code=200
+    )
+    res = get_user_context_by_email_address(reco_client, "john@acme.com")
+    assert res.outputs_prefix == "Reco.User"
+    assert res.outputs.get("email_account") != ""
```

Comment:
Lets add assertion for the email 
```suggestion
    assert res.outputs.get("email_account") != ""
    assert res.outputs.get("email_account") == "john@acme.com"
```